prometheus_api_client/metric.py

```diff
@@ -155,9 +155,10 @@ def __add__(self, other):
         """
         if self == other:
             new_metric = deepcopy(self)
-            new_metric.metric_values = new_metric.metric_values.append(
-                other.metric_values, ignore_index=True
-            )
+            new_metric.metric_values = pandas.concat([new_metric.metric_values,other.metric_values],ignore_index=True)
+            # new_metric.metric_values = new_metric.metric_values.concat(
+            #     other.metric_values, ignore_index=True
+            # )
```

Comment:
Just my 2 cents, but I think it might be cleaner to simply remove old code instead of keep it as commented out lines. But I'll defer to your and @4n4nd's judgement here :)